build.sbt

```diff
@@ -463,6 +463,8 @@ def internalProject(projectId: String, additionalSettings: sbt.Def.SettingsDefin
     )
     .settings(additionalSettings: _*)
 
+// Temporarily disabled until we get tags setup in Pekko
+/**
```

Comment:
I'll merge this - but oddly I added a v0.0.0 tag but this still fails

Reply:
https://github.com/apache/incubator-pekko-connectors/releases/tag/v0.0.0

Reply:
I see the tag, so yeah this happened because I forked the repo into my own account (`mdedetrich`) and when I cloned it onto the machine it didn't fetch upstream tags. I just fetched upstream tags and its working now.

I can either close this PR or maybe update the warning specifying what exactly to do locally?

Reply:
So I have just done some changes locally so that instead of terminating SBT it just outputs an error log, i.e.

```
<@incubator-pekko-connectors>-<⎇ disable-dynver-git-tag-check>-<±>-> sbt
[info] welcome to sbt 1.6.2 (Homebrew Java 19.0.1)
[info] loading global plugins from /Users/mdedetrich/.sbt/1.0/plugins
[info] loading settings for project incubator-pekko-connectors-build from plugins.sbt ...
[info] loading project definition from /Users/mdedetrich/github/incubator-pekko-connectors/project
[info] loading settings for project pekkoConnectors from build.sbt ...
[info] resolving key references (65251 settings) ...
[info] 
[info] ** Welcome to the sbt build definition for Pekko Connectors! **
[info] 
[info] Useful sbt tasks:
[info] 
[info]   docs/previewSite - builds Paradox and Scaladoc documentation,
[info]     starts a webserver and opens a new browser window
[info] 
[info]   test - runs all the tests for all of the connectors.
[info]     Make sure to run `docker-compose up` first.
[info] 
[info]   mqtt/testOnly *.MqttSourceSpec - runs a single test
[info] 
[info]   mimaReportBinaryIssues - checks whether this current API
[info]     is binary compatible with the released version
[info]       
[error] Failed to derive version from git tags. Maybe run `git fetch --unshallow`? Derived version: 0.0.0+2676-4ef3f6e3+20221109-1416-SNAPSHOT
[info] sbt server started at local:///Users/mdedetrich/.sbt/1.0/server/e2bc910af141664d9ebd/sock
[info] started sbt server
sbt:pekkoConnectors> 
```

@pjfanning Let me know if this is more desirable, also requested a re-review.